requirements.txt

```diff
@@ -1,7 +1,7 @@
-coverage==6.5.0 ; python_version >= "3.9" and python_version < "3.11"
-numpy==1.26.4 ; python_version >= "3.9" and python_version < "3.11"
-pandas==1.5.3 ; python_version >= "3.9" and python_version < "3.11"
-python-dateutil==2.9.0.post0 ; python_version >= "3.9" and python_version < "3.11"
-pytz==2024.1 ; python_version >= "3.9" and python_version < "3.11"
-scipy==1.12.0 ; python_version >= "3.9" and python_version < "3.11"
-six==1.16.0 ; python_version >= "3.9" and python_version < "3.11"
+coverage==6.5.0 ; python_version >= "3.9" and python_version < "4.0"
```

Comment:
Disclaimer: I have had nothing to do with this repo so far

I think giving basically a wildcard for all coming versions is a bit optimistic. You will want to run all tests on the specific Python versions you include.

How about `< 3.13`? Then you can run all tests on Python 3.11 & 3.12 and see, if the results are fine.